setup/baseConfig.yaml

```diff
@@ -31,8 +31,8 @@ foreground: true|false <run blobfuse2 in foreground or background>
 
 # Common configurations
 read-only: true|false <mount in read only mode - used for Streaming and FUSE>
-allow-other: true|false <allow other users to access the mounted directory - used for FUSE and File Cache>
-nonempty: true|false <allow mounting on non-empty directory - used for FUSE>
+allow-other: true|false <allow other users to access the mounted directory - used for FUSE and File Cache> #NOTE: Make sure user_allow_other is set in /etc/fuse.conf when set this option to true.
+#nonempty: true|false <allow mounting on non-empty directory - used for FUSE> NOTE: This function is discontinued currently. When this option specified, the value should be false always.
```

Comment:
baseconfig just lists all the possible inputs and their meanings so there is no point in commenting this out here. Also default value of each flag is false only so we can keep the comment and remove the # from here.